README.md

```diff
@@ -0,0 +1,219 @@
+
+- __Measuring stick for success:__ Can someone go out and get a better role after you’ve been their manager? You’ve got to care about the development of people to your core.
+- __Put people and the team first:__ Constantly ask what you can do to be most helpful to individuals or the team? The most helpful thing you can do may be totally different day-to-day, week-to-week, person-to-person, or team-to-team. Default to the assumption that you’re the worst on the team and treat everyone else like they’re more awesome than you. If you’ve hired right, that will be true.
+- __Constant improvement:__ Make this a fundamental value for both individuals and the team itself. Don’t set anything in stone. Don’t assume you’re ever doing something perfect. When you’ve made one improvement, move onto the next thing. Repeat forever.
+- __How to move mountains:__ Give a properly motivated team a high degree of trust and autonomy.
+- __Achieving motivation:__ Help everyone to understand the value of what the team is doing and what they as an individual are doing. Make sure everyone knows the priorities and how they were determined. Be transparent and honest about goals. Keep your reports' growth central to your focus and make it a part of their everyday life.
+- __Autonomy:__ Let teams decide for themselves how they best operate. Actively seek out new ideas. When someone has an idea, consider it with an open mind and err on the side of trying new things. You can always revert if something doesn’t work. Don’t micromanage someone’s day or their work.
+- __Create a safe and comfortable psychological space__: Strive to craft this work environment for engineers and teams. Encourage people to share their ideas. Listen to those opinions and feedback. Take action on what you’re hearing. Foster a culture of blameless criticism.
+- __Management is important and valuable__: Great managers can make a positive difference in people's lives and can materially impact the success of a team or a business. Bad management and lack of management can be equally destructive. Try to provide a great experience to everyone you work with. An important distinction here is to not focus on trying to make everyone like you all the time. Great management often means telling people hard truths. Which leads into...
+- __Feedback is vital:__ Give pointed, direct feedback on a regular basis. Don’t wait for an annual review to help someone improve. Try to generalize negative feedback and be specific with positive feedback. Additionally, seek out feedback from those you work with, in all directions, in order to drive your own growth.
+- __Diverse teams make the best decisions__: This may seem obvious but software industry demographics indicate it is not. Seek out a variety of backgrounds and talent profiles when building a team. Strive for great communication to unlock the power of these different experiences.
```

Comment:
I'm going to weigh how much I want to elaborate on this. This is another topic that could warrant an essay-length explanation (and better writers have already done this) to do it justice and I'm not confident this is the right medium.

My intent here is to articulate that I'm open, even desirous, of building teams that are comprised of people from not just a variety of demographics but a variety of backgrounds before they got into software. Examples are people who went to boot camps rather than CS programs, were non-CS majors (like me!), or got their careers started in other fields even if they have a CS degree.